static/js/components/PersonalForm.js

```diff
@@ -39,6 +40,28 @@ export default class PersonalForm extends ProfileFormFields {
     ui:                     UIState,
   };
 
+  renderRomanizedFields = () => {
+    const { profile } =  this.props;
+    if (showRomanizedFields(profile)) {
+      return (
+        <Cell col={12}>
+          <section className="romanized-name">
+            <h3>Plese enter your name in Latin characters</h3>
```

Comment:
The CP-1252 character set is hard to describe, so I think we should avoid trying to be overly specific here. Just use the text in the mock.